src/main/java/core/basesyntax/color/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.color;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random RANDOM = new Random(System.currentTimeMillis());
```

Comment:
```suggestion
    private final Random radnom = new Random();
```
There is no need to make this variable a constant